tracks/ruby/exercises/resistor-colors/mentoring.md

```diff
@@ -0,0 +1,33 @@
+Resistor Colors is a side exercise, unlocked by Hello World
+
+### New Concepts
+Array, Array#index, `map`, Constant, chaining methods
+
+### Minimal solution for approval
+```ruby
+
+class ResistorColors
+
+  COLOR_CODES = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"]
```

Comment:
```suggestion
  COLOR_CODES = %w[black brown red orange yellow green blue violet grey white].freeze
```

Default style via [rubocop WordArray](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/WordArray) and [rubocop MutableConstant](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/MutableConstant). 

Add below this solution that:
- "Alternatives include writing out the array using brackets"; the talking points already mention this.
- `freeze` should only be a talking point but not a reason to disapprove